### ISSUE TITLE
Enable alert generation – creation of a new AlertDefinition object

### ISSUE THREAD

**Issue by ivandank12** [SamCooper/MC_SPEC_RIDS#204]
The specification states that the Enable alert generation operation shall create a new Alert Definition object. 
For clarity, the specification should also state that the new AlertDefinition object shall be the current AlertDefinition used for the specific AlertIdentity.
Also it would be cleared if the identifiers for the new objects are returned.

**Comment by SamCooper** [comment]
It does say that:

k)  If a new AlertDefinition object is created then that new object shall be the current AlertDefinition used for the specific AlertIdentity.

**Comment by ivandank12** [comment]
Agreed it does already specify that. The issue that I should have raised is that if a new definition object is created, the definition identifier should be returned. This would be consistent with the other operations such as update definition.

**Comment by SamCooper** [comment]
All enabled should be like this then. At this point I'm not going to make such a big change so I'm going to mark this as for issue 2 of the specification.

**Comment by ivandank12** [comment]
OK agreed

**Comment by SamCooper** [comment]
Don't close it!
